cmd/trace/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/crossplaneio/crossplane-cli/pkg/trace"
+
+	"k8s.io/client-go/discovery/cached/disk"
+
+	"k8s.io/client-go/restmapper"
+
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+
+func main() {
+	var help bool
+	var kubeconfig string
+	var namespace string
+	var outputFormat string
+	if home := homedir.HomeDir(); home != "" {
+		pflag.StringVar(&kubeconfig, "kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) Absolute path to the kubeconfig file")
```

Comment:
I don't know enough about CLI libraries in go enough to have an opinion here, but is this the library that kubectl uses? I would default to trying to use the same stuff as kubectl.